src/dbnode/generated/proto/namespace/namespace.proto

```diff
@@ -33,6 +33,41 @@ message NamespaceOptions {
     bool coldWritesEnabled                          = 10;
     NamespaceRuntimeOptions runtimeOptions          = 11;
     google.protobuf.BoolValue cacheBlocksOnRetrieve = 12;
+    AggregationOptions aggregationOptions           = 13;
+}
+
+// AggregationOptions is a set of options for aggregating data
+// within the namespace.
+message AggregationOptions {
+    // aggregations is a repeated field to support the ability to send aggregated data
+    // to a namespace also receiving unaggregated data. In this case, the namespace will
+    // have one Aggregation with aggregated set to false and another with aggregated set to true.
+    repeated Aggregation aggregations = 1;
+}
+
+// Aggregation describes data points within the namespace.
+message Aggregation {
```

Comment:
Re-worked this to remove AggregationType enum and use a boolean. Also grouped all options specific to being aggregated into a a specific field of `AggregatedAttributes` types which makes it more clear that it doesn't apply when `aggregated = false`

Reply:
nice iteration.